diff --git a/tests/auto/qml/ecmascripttests/test262.py b/tests/auto/qml/ecmascripttests/test262.py
index 19551e3b..fb06e9ef 100755
--- a/tests/auto/qml/ecmascripttests/test262.py
+++ b/tests/auto/qml/ecmascripttests/test262.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 #############################################################################
 ##
 ## Copyright (C) 2017 The Qt Company Ltd.
diff --git a/tests/auto/qml/ecmascripttests/test262/make.py b/tests/auto/qml/ecmascripttests/test262/make.py
index 6594c5c5..c8ed7587 100755
--- a/tests/auto/qml/ecmascripttests/test262/make.py
+++ b/tests/auto/qml/ecmascripttests/test262/make.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # Copyright (C) 2016 the V8 project authors. All rights reserved.
 # This code is governed by the BSD license found in the LICENSE file.
 
diff --git a/tests/auto/qml/ecmascripttests/test262/tools/generation/generator.py b/tests/auto/qml/ecmascripttests/test262/tools/generation/generator.py
index 64a0ccf0..dda1d92f 100755
--- a/tests/auto/qml/ecmascripttests/test262/tools/generation/generator.py
+++ b/tests/auto/qml/ecmascripttests/test262/tools/generation/generator.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # Copyright (C) 2016 the V8 project authors. All rights reserved.
 # This code is governed by the BSD license found in the LICENSE file.
 
diff --git a/tests/auto/qml/ecmascripttests/test262/tools/generation/test/run.py b/tests/auto/qml/ecmascripttests/test262/tools/generation/test/run.py
index c21efa1b..e86e59b5 100755
--- a/tests/auto/qml/ecmascripttests/test262/tools/generation/test/run.py
+++ b/tests/auto/qml/ecmascripttests/test262/tools/generation/test/run.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # Copyright (C) 2016 the V8 project authors. All rights reserved.
 # This code is governed by the BSD license found in the LICENSE file.
 
diff --git a/tests/auto/qml/ecmascripttests/test262/tools/lint/lint.py b/tests/auto/qml/ecmascripttests/test262/tools/lint/lint.py
index 979616a5..e1aff4df 100755
--- a/tests/auto/qml/ecmascripttests/test262/tools/lint/lint.py
+++ b/tests/auto/qml/ecmascripttests/test262/tools/lint/lint.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # Copyright (C) 2017 Mike Pennisi. All rights reserved.
 # This code is governed by the BSD license found in the LICENSE file.
 
diff --git a/tests/auto/qml/ecmascripttests/test262/tools/lint/test/run.py b/tests/auto/qml/ecmascripttests/test262/tools/lint/test/run.py
index 9c83c794..f17f3892 100755
--- a/tests/auto/qml/ecmascripttests/test262/tools/lint/test/run.py
+++ b/tests/auto/qml/ecmascripttests/test262/tools/lint/test/run.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # Copyright (C) 2017 Mike Pennisi. All rights reserved.
 # This code is governed by the BSD license found in the LICENSE file.
 
diff --git a/tests/auto/qml/ecmascripttests/test262/tools/packaging/monkeyYaml.py b/tests/auto/qml/ecmascripttests/test262/tools/packaging/monkeyYaml.py
index 21c3fa42..21f0a236 100644
--- a/tests/auto/qml/ecmascripttests/test262/tools/packaging/monkeyYaml.py
+++ b/tests/auto/qml/ecmascripttests/test262/tools/packaging/monkeyYaml.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # Copyright 2014 by Sam Mikes.  All rights reserved.
 # This code is governed by the BSD license found in the LICENSE file.
 
diff --git a/tests/auto/qml/ecmascripttests/test262/tools/packaging/parseTestRecord.py b/tests/auto/qml/ecmascripttests/test262/tools/packaging/parseTestRecord.py
index 681039d3..2a3ffb2c 100644
--- a/tests/auto/qml/ecmascripttests/test262/tools/packaging/parseTestRecord.py
+++ b/tests/auto/qml/ecmascripttests/test262/tools/packaging/parseTestRecord.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 
 # Copyright 2011 by Google, Inc.  All rights reserved.
 # This code is governed by the BSD license found in the LICENSE file.
diff --git a/tests/auto/qml/ecmascripttests/test262/tools/packaging/test/test_common.py b/tests/auto/qml/ecmascripttests/test262/tools/packaging/test/test_common.py
index 0d6600b7..7b2e36a3 100644
--- a/tests/auto/qml/ecmascripttests/test262/tools/packaging/test/test_common.py
+++ b/tests/auto/qml/ecmascripttests/test262/tools/packaging/test/test_common.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 
 # Copyright 2014 by Sam Mikes.  All rights reserved.
 # This code is governed by the BSD license found in the LICENSE file.
diff --git a/tests/auto/qml/ecmascripttests/test262/tools/packaging/test/test_monkeyYaml.py b/tests/auto/qml/ecmascripttests/test262/tools/packaging/test/test_monkeyYaml.py
index 92d4e613..3b76a9bc 100644
--- a/tests/auto/qml/ecmascripttests/test262/tools/packaging/test/test_monkeyYaml.py
+++ b/tests/auto/qml/ecmascripttests/test262/tools/packaging/test/test_monkeyYaml.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 
 # Copyright 2014 by Sam Mikes.  All rights reserved.
 # This code is governed by the BSD license found in the LICENSE file.
diff --git a/tests/auto/qml/ecmascripttests/test262/tools/packaging/test/test_parseTestRecord.py b/tests/auto/qml/ecmascripttests/test262/tools/packaging/test/test_parseTestRecord.py
index 7d1e9a64..689f1211 100644
--- a/tests/auto/qml/ecmascripttests/test262/tools/packaging/test/test_parseTestRecord.py
+++ b/tests/auto/qml/ecmascripttests/test262/tools/packaging/test/test_parseTestRecord.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 
 # Copyright 2014 by Sam Mikes.  All rights reserved.
 # This code is governed by the BSD license found in the LICENSE file.
